### ISSUE TITLE
Interceptor.detach() exposure to JS

### ISSUE THREAD

**Issue by CarrotCultivator** [frida/frida#3479]
Hello. 
I noticed that there is internal functionality that is allows for detaching of a certain hook from Interceptor.attach(), namely at
https://github.com/frida/frida-gum/blob/main/bindings/gumjs/gumv8interceptor.cpp#L674 . It would be useful if there was an Interceptor.detach(hooked_address), available to Javascript.
Interceptor.revert() does not seem to work for attach()-ed hooks.

One use case: When we hook multiple addresses, the console.log() output can be massive, especially when each hook is called multiple times. But we  may only need the log to be printed once for each hook. The simplest way is to detach().
